src/SoapClientPlus.php

```diff
@@ -310,32 +310,12 @@ public function resetDebugValue(): void
      * @return mixed
      * @deprecated
      */
-    public function __call(string $name, $args): mixed
+    public function __call(string $name, array $args): mixed
     {
         array_unshift($args, $name);
         return call_user_func_array([$this, '__soapCall'], $args);
     }
 
-    /**
-     * __soapCall overload
-     *
-     * @param string $name
-     * @param string|array $args
-     * @param array $options
-     * @param array $inputHeaders
-     * @param array $outputHeaders
-     * @return mixed
-     * @throws \Exception
-     */
-    public function __soapCall(string $name, string|array $args, $options = [], $inputHeaders = [], &$outputHeaders = []): mixed
-    {
-        if (is_string($args)) {
-            $args = $this->createArgumentArrayFromXML($args, $name);
-        }
-
-        return parent::__soapCall($name, $args, $options, $inputHeaders, $outputHeaders);
-    }
-
```

Comment:
Why remove this?  It's still present in the class: https://www.php.net/manual/en/soapclient.soapcall.php